satzung.md

```diff
@@ -63,60 +63,71 @@ Die Organe der Fachschaft sind:
 (3) Die in (2) festgelegten Ämter können durch Beschluss mit einer Zwei-Drittel-Mehrheit auch innerhalb einer Amtszeit neu vergeben werden.
 
 
-## §5b. Aufgabenverteilung
+## §4c Aufgabenverteilung
 
-(1) Der Vorsitzende des Fachschaftsrates ist verantwortlich für die Einberufung der Sitzungen, Verwaltung der Tagesordnung und Festlegung der Protokollführung. Er ist ohne weiteren Beschluss automatisch ebenfalls Beauftragter für die Vernetzung mit den anderen Organen der Studierendenschaft laut Satzung der Studierendenschaft der Universität Potsdam.
+(1) Der Vorsitzende des Fachschaftsrates ist verantwortlich für die Sitzungsleitung. Er ist ohne weiteren Beschluss automatisch ebenfalls Beauftragter für die Vernetzung mit den anderen Organen der Studierendenschaft laut Satzung der Studierendenschaft der Universität Potsdam.
```

Comment:
Ich glaube wir sollten noch genauer klären, was das Wort Sitzungsleitung genau bedeutet, oder? Also man könnte eigentlich ja das alte stehen lassen und dann in Klammern einfach dahinter schreiben, dass das die Sitzungsleitung ist.